tests/model_serving/model_server/multi_node/conftest.py

```diff
@@ -61,28 +69,47 @@ def models_bucket_downloaded_model_data(
 
 
 @pytest.fixture(scope="class")
-def multi_node_inference_service(
+def multi_node_serving_runtime(
     request: FixtureRequest,
     admin_client: DynamicClient,
     model_namespace: Namespace,
-    serving_runtime_from_template: ServingRuntime,
+) -> Generator[ServingRuntime, Any, Any]:
+    with ServingRuntimeFromTemplate(
+        client=admin_client,
+        name="vllm-multinode-runtime",  # TODO: rename servingruntime when RHOAIENG-16147 is resolved
+        namespace=model_namespace.name,
+        template_name="vllm-multinode-runtime-template",
+        multi_model=False,
+        enable_http=True,
+    ) as model_runtime:
+        yield model_runtime
+
+
+@pytest.fixture(scope="class")
+def multi_node_inference_service(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
+    multi_node_serving_runtime: ServingRuntime,
     model_pvc: PersistentVolumeClaim,
     models_bucket_downloaded_model_data: str,
 ) -> Generator[InferenceService, Any, Any]:
     with create_isvc(
         client=admin_client,
         name=request.param["name"],
-        namespace=model_namespace.name,
-        runtime=serving_runtime_from_template.name,
+        namespace=multi_node_serving_runtime.namespace,
+        runtime=multi_node_serving_runtime.name,
         storage_uri=f"pvc://{model_pvc.name}/{models_bucket_downloaded_model_data}",
-        model_format=serving_runtime_from_template.instance.spec.supportedModelFormats[0].name,
+        model_format=multi_node_serving_runtime.instance.spec.supportedModelFormats[0].name,
         deployment_mode=KServeDeploymentType.RAW_DEPLOYMENT,
         autoscaler_mode="external",
         multi_node_worker_spec={},
         wait_for_predictor_pods=False,
     ) as isvc:
         wait_for_inference_deployment_replicas(
-            client=admin_client, isvc=isvc, expected_num_deployments=2, runtime_name=serving_runtime_from_template.name
+            client=admin_client,
+            isvc=isvc,
+            expected_num_deployments=2,
```

Comment:
Is this dependent on something? Can a constant be used?

Reply:
> Is this dependent on something?
the logic of the feature but nothing to check against